packages/transaction_signing_gateway/test/aes_cipher_test.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:transaction_signing_gateway/encrypt/aes_cipher.dart';
+
+void main() {
```

Comment:
@alain2sf  here are some tests to prove encrypting/decrypting works and that we use random salt on each encryption